app/models/concerns/blacklight/document/email.rb

```diff
@@ -9,7 +9,6 @@ module Email
       def to_email_text
         body = []
         add_bibliographic_text(body)
-        add_holdings_text(body)
```

Comment:
Removing the Blacklight 7-specific holdings display code, so that we can display the view component in either Blacklight 7 or 8.